one_fm/fixtures/workflow.json

```diff
@@ -1,4 +1,160 @@
+    "parent": "Subcontract Staff Shortlist Application",
+    "parentfield": "states",
+    "parenttype": "Workflow",
+    "state": "Rejected",
+    "update_field": null,
+    "update_value": null
+   },
+   {
+    "allow_edit": "Onboarding Officer",
+    "doc_status": "2",
+    "is_optional_state": 0,
+    "message": null,
+    "next_action_email_template": null,
+    "parent": "Subcontract Staff Shortlist Application",
+    "parentfield": "states",
+    "parenttype": "Workflow",
+    "state": "Cancelled",
+    "update_field": null,
+    "update_value": null
+   }
+  ],
+  "transitions": [
+   {
+    "action": "Submit for Review",
+    "allow_self_approval": 1,
+    "allowed": "Operations Supervisor",
+    "allowed_user_field": null,
+    "allowed_user_id": null,
+    "condition": null,
+    "next_state": "Pending Onboarding Officer",
+    "parent": "Subcontract Staff Shortlist Application",
+    "parentfield": "transitions",
+    "parenttype": "Workflow",
+    "skip_creation_of_workflow_action": 0,
+    "skip_multiple_action": 0,
+    "state": "Draft"
+   },
+   {
+    "action": "Approve",
+    "allow_self_approval": 1,
+    "allowed": "Onboarding Officer",
+    "allowed_user_field": null,
+    "allowed_user_id": null,
+    "condition": null,
+    "next_state": "Approved",
+    "parent": "Subcontract Staff Shortlist Application",
+    "parentfield": "transitions",
+    "parenttype": "Workflow",
+    "skip_creation_of_workflow_action": 0,
+    "skip_multiple_action": 0,
+    "state": "Pending Onboarding Officer"
+   },
+   {
+    "action": "Return",
+    "allow_self_approval": 1,
+    "allowed": "Onboarding Officer",
+    "allowed_user_field": null,
+    "allowed_user_id": null,
+    "condition": null,
+    "next_state": "Draft",
+    "parent": "Subcontract Staff Shortlist Application",
+    "parentfield": "transitions",
+    "parenttype": "Workflow",
+    "skip_creation_of_workflow_action": 0,
+    "skip_multiple_action": 0,
+    "state": "Pending Onboarding Officer"
+   },
+   {
+    "action": "Reject",
+    "allow_self_approval": 1,
+    "allowed": "Onboarding Officer",
+    "allowed_user_field": null,
+    "allowed_user_id": null,
+    "condition": null,
+    "next_state": "Rejected",
+    "parent": "Subcontract Staff Shortlist Application",
+    "parentfield": "transitions",
+    "parenttype": "Workflow",
+    "skip_creation_of_workflow_action": 0,
+    "skip_multiple_action": 0,
+    "state": "Pending Onboarding Officer"
+   },
+   {
+    "action": "Cancel",
+    "allow_self_approval": 1,
+    "allowed": "Onboarding Officer",
+    "allowed_user_field": null,
+    "allowed_user_id": null,
+    "condition": null,
+    "next_state": "Cancelled",
+    "parent": "Subcontract Staff Shortlist Application",
+    "parentfield": "transitions",
+    "parenttype": "Workflow",
+    "skip_creation_of_workflow_action": 0,
+    "skip_multiple_action": 0,
+    "state": "Approved"
+   }
+  ],
+  "workflow_name": "Subcontract Staff Shortlist Application",
```

Comment:
@AbrahamAdekunle You have changes from other Workflow Doctypes. Could you confirm if it's linked to this story?